.github/actions/integration-test-setup/action.yml

```diff
@@ -14,6 +14,10 @@ inputs:
 runs:
   using: composite
   steps:
+    - id: update-hosts
+      name: Update /etc/hosts
+      uses: ./.github/actions/update-hosts
```

Comment:
Not sure if we need to use this somewhere else too in the UI/JS tests? CC @jonkoops

Reply:
No, those should be fine without.